src/assets/js/irma-functions.js

```diff
@@ -26,7 +28,7 @@ function irmaDiscloseOrSign(attributes, header = 'Disclosing attribute with', la
     });
 }
 
-function irmaIssueCredential(credential, attributes, header = 'Issuing credential with', disclosePayload = null) {
+function irmaIssueCredential(credential, attributes, header = 'Issuing credential with', disclosePayload = null, revocationKey = None) {
```

Comment:
`revocationKey = null`?

Reply:
Hum, I think would be better to have a specific method called `irmaIssueRevocableCredential` instead, as this could be more meaningful.

Reply:
Cool, I've separated the `irmaIssueRevocableCredential` and `irmaIssueCredential` functions, but `IrmaDiscloseOrSign` is still handling both the original disclosure and disclosure with revocation. Does that sound sensible?

Reply:
Also what part of the front end needs to be changed to use this? Or should we hold off making those changes until we know revocation is working?

Reply:
I agree that disclosure or sign seems to fit better into a single function.